v4/source/posts.yaml

```diff
@@ -417,6 +417,48 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/users/{user_id}/channels/{channel_id}/posts/unread':
+    get:
+      tags:
+        - posts
+      summary: Get posts around last unread
+      description: |
+        Get posts around last unread post by a user in a channel. Use the query parameters `limit_before` and `limit_after` to modify the number of posts returned from this endpoint.
+        ##### Permissions
+        Must be logged in as the user or have `edit_other_users` permission, and must have `read_channel` permission for the channel.
+      parameters:
+        - name: user_id
+          in: path
+          description: ID of the user
+          required: true
+          type: string
+        - name: channel_id
+          in: path
+          description: The channel ID to get the posts for
+          required: true
+          type: string
+        - name: limit_before
```

Comment:
Is there a maximum `limit_before` and `limit_after`?

Reply:
Yes, [max is 200](https://github.com/mattermost/mattermost-server/blob/master/web/params.go#L22).  I'll add that in description.

Reply:
Description added. Please take a look again, thanks